corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/content.less

```diff
@@ -73,6 +73,50 @@ body {
   height: 100%;
 }
 
+.marker-pin {
+  width: 30px;
+  height: 30px;
+  border-radius: 50% 50% 50% 0;
+  background: #c30b82;
+  position: absolute;
+  transform: rotate(-45deg);
+  left: 50%;
+  top: 50%;
+  margin: -15px 0 0 -15px;
+}
+// to draw white circle
+.marker-pin::after {
```

Comment:
I'm generally a fan of nesting LESS, so I'd put this inside the `marker-pin` block above, though you can also say that's just a stylistic difference.